src/pre_process/m_initial_condition.fpp

```diff
@@ -284,6 +288,9 @@ contains
                 elseif (patch_ib(i)%geometry == 4) then
                     call s_airfoil(i, ib_markers%sf, q_prim_vf, .true.)
                     call s_compute_airfoil_levelset(levelset, levelset_norm, i)
+                    ! STL+IBM patch
+                elseif (patch_ib(i)%geometry == 5) then
+                    call s_model(i, ib_markers%sf, q_prim_vf, .true., levelset, levelset_norm)
```

Comment:
see above comment about putting a raw boolean in the middle of the argument list.